README.md

```diff
@@ -12,85 +12,162 @@ providers, GCE has a couple of advantages for Chef cookbook testing:
+to the appropriate user's `~/.ssh/authorized_keys` file when Chef first
+connects to perform the bootstrap process.
+
+ * If you don't have one, create a key using `ssh-keygen`
+ * Log in to the GCP console, select your project, go to Compute Engine, and go to the Metadata tab.
+ * Select the "SSH Keys" tab.
+ * Add a new item, and paste in your public key.
+    * Note: to change the username automatically detected for the key, prefix your key with the username
+      you plan to use to connect to a new instance. For example, if you plan to connect
+      as "chefuser", your key should look like: `chefuser:ssh-rsa AAAAB3N...`
+ * Click "Save".
+
+Alternatively, the Google Cloud SDK (a.k.a. `gcloud`) will create a SSH key
+for you when you create and access your first instance:
+
+ 1. Create a small test instance:
+    `gcloud compute instances create instance1 --zone us-central1-f --image debian-8 --machine-type g1-small`
+ 1. Ensure your SSH keys allow access to the new instance
+    `gcloud compute ssh instance1 --zone us-central1-f`
+ 1. Log out and delete the instance
+    `gcloud compute instances delete instance1 --zone us-central1-f`
+
+You will now have a local SSH keypair, `~/.ssh/google_compute_engine[.pub]` that
+will be used for connecting to your GCE Linux instances for this local username
+when you use the `gcloud compute ssh` command. You can tell Test Kitchen to use
+this key by adding it to the transport section of your .kitchen.yml:
+
+```yaml
+transport:
+  ssh_key:
+    - ~/.ssh/google_compute_engine
+```
+
+You can find [more information on configuring SSH keys](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys) in
+the Google Compute Engine documentation.
 
 ## Installation
 
-Assuming you are using Bundler, ensure the Gemfile within your Chef
-cookbook contains at least the following:
+Install the gem with:
 
-```ruby
-source 'https://rubygems.org'
+```sh
+gem install kitchen-google
+```
+
+Or, even better, if you're using the ChefDK:
 
-gem 'berkshelf'
+```sh
+chef gem install kitchen-google
+```
+
+If you're using Bundler, simply add it to your Gemfile:
 
-group :integration do
-  gem 'kitchen-gce'
-end
+```ruby
+gem "kitchen-google", "~> 1.0"
 ```
 
-Then, execute `bundle install`.
+... and then run `bundle install`.
+
 
 ## Configuration
 
-### `area`
+### `project`
 
-Deprecated - use equivalent [`region`](#region) instead.  If both `area` and
-`region` are set, the value of `region` will be used.
+**Required**. The project ID of the GCP project into which Test Kitchen
+instances will be launched. This can be found on the "Manage All Projects"
+screen, found under the "Select a Project" drop-down at the top of the
+GCP console.
 
-### `autodelete_disk`
+Note that this parameter requires the "Project ID", not the "Project Name."
 
-Boolean specifying whether or not to automatically delete boot disk
-for test instance.  Default: `true`
+Example: "mycompany-test"
```

Comment:
Autogen names look closer to something like: dev-utility-94444
I would use a default-looking autogenerated project name that looks similar, like dev-utility-xxxxx

This may make it a bit confusing for newcomers - what do you think?
Ditto for example.